apps/camera/js/views/setting.js

```diff
@@ -44,34 +44,37 @@ module.exports = View.extend({
     // The settings list is a listbox (list of actionable items) thus the
     // setting must be an 'option'.
     this.el.setAttribute('role', 'option');
-    // The only way to exclude content from :before element (present in setting
-    // item) is to override it with ARIA label.
-    this.l10n.setAttributes(this.el, 'setting-option-' + data.title, {
-      value: this.localizeValue(data)
-    });
-
     this.el.innerHTML = this.template(data);
 
+    if (data.optionsLocalizable === false) {
+      this.l10n.setAttributes(
+        this.el.querySelector('.setting_title'),
+        'setting-option-' + data.title, {
+          value: data.value
+        }
+      );
+      this.el.querySelector('.setting_value').textContent = data.value;
+    } else {
+      this.l10n.formatValue(data.value).then((value) => {
+        this.l10n.setAttributes(this.el, 'setting-option-' + data.title, {
+          value: value
+        });
+      });
+      this.l10n.setAttributes(
+        this.el.querySelector('.setting_value'), data.value);
+    }
```

Comment:
This is lookinga bit messy. I think this would be more elegant if you were able to prepare the attribute strings (eg. `data.titlel10n = ...`) and then pass them into the `.template()` function.